### DISCUSSION TITLE
Fetching Channel details of a product that was created under from shopApi

### DISCUSSION THREAD

**Discussion by ramkishore-yerubandi** [vendurehq/vendure#1255] [Q&A] [answered]
Is there a way to fetch Channel details that the product was created from shop Api? I searched in the documentation under shop-API but could not find any reference on this.

**Comment by michaelbromley** [comment, marked as answer]
No, not by default. You'll need to implement an extension to the schema, either a new query or a new field on the Product type: https://www.vendure.io/docs/plugins/plugin-examples/extending-graphql-api/
